### PR TITLE
IGP 15

### PR DESCRIPTION
**Pull request by thrilok209**

### CODE REVIEW

File: contracts/payloads/IGP15/PayloadIGP15.sol [hunk 1]
```diff
@@ -0,0 +1,668 @@
...
+        AdminModuleStructs.RateDataV2Params[]
+            memory params_ = new AdminModuleStructs.RateDataV2Params[](1);
+
+        params_[0] = AdminModuleStructs.RateDataV2Params({
+            token: sUSDe_ADDRESS, // sUSDe
+            kink1: 50 * 1e2, // 50%
+            kink2: 80 * 1e2, // 80%
+            rateAtUtilizationZero: 0, // 0%
+            rateAtUtilizationKink1: 20 * 1e2, // 20%
+            rateAtUtilizationKink2: 40 * 1e2, // 40%
```
Comment (bergben) [inline contracts/payloads/IGP15/PayloadIGP15.sol:417]: proposal says 10% for rateAtUtilizationKink1, then 20% and 50% here? https://atlas.instadapp.io/proposals/15

File: contracts/payloads/IGP15/PayloadIGP15.sol [hunk 2]
```diff
@@ -0,0 +1,668 @@
...
+    }
+
+    /// @notice Action 2: Set token config for sUSDe on Liquidity.
+    function action2() internal {
+        AdminModuleStructs.TokenConfig[]
+            memory params_ = new AdminModuleStructs.TokenConfig[](1);
+
+        params_[0] = AdminModuleStructs.TokenConfig({
+            token: sUSDe_ADDRESS, // sUSDe
+            threshold: 0.3 * 1e2, // 0.3
```
Comment (bergben) [inline contracts/payloads/IGP15/PayloadIGP15.sol:431]: 0.25 according to proposal (https://atlas.instadapp.io/proposals/15)

File: contracts/payloads/IGP15/PayloadIGP15.sol [hunk 3]
```diff
@@ -0,0 +1,668 @@
...
+        // Set user supply config for the vault on Liquidity Layer.
+        {
+            AdminModuleStructs.UserSupplyConfig[]
+                memory configs_ = new AdminModuleStructs.UserSupplyConfig[](1);
+
+            configs_[0] = AdminModuleStructs.UserSupplyConfig({
+                user: address(vault_),
+                token: sUSDe_ADDRESS,
+                mode: 1,
+                expandPercent: 25 * 1e2, // 25%
```
Comment (bergben) [inline contracts/payloads/IGP15/PayloadIGP15.sol:521]: in forum post this is 20%. although that might be wrong if we keep our standard as used for other vaults of 25%. https://gov.instadapp.io/t/launch-susde-usdc-and-susde-usdt/713

Comment (bergben) [reply contracts/payloads/IGP15/PayloadIGP15.sol:521]: same for USDT

File: contracts/payloads/IGP15/PayloadIGP15.sol (hunk 1 above)
Comment (KaymasJain) [reply contracts/payloads/IGP15/PayloadIGP15.sol:417]: Yes! Misplaced it. Although, doesn't matter as this curve is not being used.

File: contracts/payloads/IGP15/PayloadIGP15.sol (hunk 2 above)
Comment (KaymasJain) [reply contracts/payloads/IGP15/PayloadIGP15.sol:431]: Yes, should have been according to the proposal.

File: contracts/payloads/IGP15/PayloadIGP15.sol (hunk 3 above)
Comment (KaymasJain) [reply contracts/payloads/IGP15/PayloadIGP15.sol:521]: It's effectively 20%. With withdrawal gap!

**Pull request merged**